Sorting/3_way_quick_sort.cpp

```diff
@@ -0,0 +1,93 @@
+#include<bits/stdc++.h>
```

Comment:
Don't use bits/stdc++.h because this is quite Linux specific and slows down the compiler process.